skypy/galaxies/stellar_mass.py

```diff
@@ -74,3 +75,58 @@ def schechter_smf_mass(redshift, alpha, m_star, m_min, m_max, size=None,
     m = schechter(alpha, x_min, x_max, resolution, size=size, scale=m_star)
 
     return m
+
+
+def schechter_smf_parameters(active_parameters, fsatellite, fenvironment):
+    r'''Schechter parameters.
+    This function returns the Schechter mass function parameters
+    for active galaxies (centrals and satellites)
+    and passive galaxies (mass- and satellite-quenched)
+    based on equation (15) in de la Bella et al. 2021 [1]_.
+
+    Parameters
+    ----------
+    active_parameters: tuple
+        Schechter mass function parameters for the entire active
+        sample of galaxies: :math:`(\phi_b, \alpha_b, m_{*})`.
```

Comment:
I suggest you call the parameter phi_{*,b}. I know it does not make a real difference but then it is consistent equation (15) in the paper.
```suggestion
        sample of galaxies: :math:`(\phi_{*,b}, \alpha_b, m_{*})`.
```